code-for-canada/project-thundercat
# [Delete Response Confirmation] Added delete confirmation popup for email response and task

# Description

I used the _PopupBox.jsx_ and _SystemMessage.jsx_ components to create a delete confirmation dialog when you want to delete an email response and / or a task.

## Type of change

- New feature (non-breaking change which adds functionality)

## Screenshot

**Email Response Delete Confirmation:**

![image](https://user-images.githubusercontent.com/23021242/56207209-2eadbe80-601c-11e9-9c0c-891eeeb6e91d.png)

![image](https://user-images.githubusercontent.com/23021242/56207239-3cfbda80-601c-11e9-9d3b-767a679444ba.png)

**Task Delete Confirmation:**

![image](https://user-images.githubusercontent.com/23021242/56207276-543ac800-601c-11e9-9e4f-c9592051fd52.png)

![image](https://user-images.githubusercontent.com/23021242/56207290-5c930300-601c-11e9-86d5-478d292ddf81.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Complete the _pre-test_ process.
3.  Open the _Inbox_ tab.
4.  Add an email response and a task.
5.  Try to delete both of them and make sure that the popup dialog is looking good and working well, meaning that it does what it is supposed to do.

# Checklist

Applicable for all code changes.
